src/widget/wlabel.cpp

```diff
@@ -63,6 +65,34 @@ void WLabel::setup(QDomNode node, const SkinContext& context) {
             setAlignment(Qt::AlignLeft | Qt::AlignVCenter);
         }
     }
+    // Ellipsis
+    if (context.hasNode(node, "Ellipsis")) {
```

Comment:
It's been our style to mirror the QT naming as much as possible, so we should call the tag Elide.  And we should accept middle, but not center.  And "none" should be an explicitly handled option since the enum exists.